src/hotspot/os_cpu/linux_ppc/os_linux_ppc.cpp

```diff
@@ -517,8 +517,8 @@ int os::extra_bang_size_in_bytes() {
   return 0;
 }
 
-#ifdef HAVE_FUNCTION_DESCRIPTORS
-void* os::Linux::resolve_function_descriptor(void* p) {
+#if !defined(ABI_ELFv2)
```

Comment:
I don't think many people will be able to map this to any specific platform. At a minimum some comment is needed to explain this.